README.md

```diff
@@ -58,10 +59,11 @@ g CTRL+] - pop up a selector if there is more than one definition; if there is o
 \rt - rebuild tags
 ```
 
-Git Grep - Fugitive
+Git - Fugitive
 
 ```
 \gw - git grep for the word under the cursor
+\gh - get the github url for the current file
```

Comment:
Neither of these use Fugitive, so maybe the heading should just be `Git`?

Reply:
Oops, I meant to change that and neglected to.  Fixed